solr/solrj/src/java/org/apache/solr/client/solrj/request/CollectionAdminRequest.java

```diff
@@ -1864,7 +1864,8 @@ public RequestStatusState waitFor(SolrClient client, long timeoutSeconds)
       long finishTime = System.nanoTime() + TimeUnit.SECONDS.toNanos(timeoutSeconds);
       RequestStatusState state = RequestStatusState.NOT_FOUND;
       while (System.nanoTime() < finishTime) {
-        state = this.process(client).getRequestStatus();
+        RequestStatusResponse response = this.process(client);
```

Comment:
I just wanted this on its own line so that in a debugger we can easily see what the response payload has